pages/index.js

```diff
@@ -0,0 +1,357 @@
+import Head from "next/head";
+import { useEffect, useState } from "react";
+import styles from "../styles/Home.module.css";
+import Header from "../components/Header";
+import ClientLibraries from "../components/ClientLibraries";
+
+export default function Home() {
+  const [latestPosts, setLatestPosts] = useState();
+
+  const fetchDevToPosts = async () => {
+    const data = await fetch(
+      "https://dev.to/api/articles?username=stripe&state=all"
+    );
+    const posts = await data.json();
+
+    const latest = posts.slice(0, 3);
+
+    setLatestPosts(latest);
+  };
+
+  useEffect(() => {
+    fetchDevToPosts();
+  }, []);
+
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>Stripe Developer Resources</title>
+        <link
+          href="https://fonts.googleapis.com/css?family=Source+Code+Pro"
+          rel="stylesheet"
+        />
+        <link rel="shortcut icon" href="favicon.ico" />
+        <meta charset="utf-8" />
```

Comment:
one more console error, apparently the React-y way to do this is `charSet="utf-8"`

<img width="649" alt="image" src="https://user-images.githubusercontent.com/102179546/189415366-db0e6120-7b7f-44df-964e-208ab136db80.png">